skill-tree /pom.xml

```diff
@@ -1,42 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.1.0</version>
+		<version>3.1.2</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>com.RDS</groupId>
-	<artifactId>skill-tree </artifactId>
+	<artifactId>skill-tree</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<name>skill-tree </name>
+	<name>skill-tree</name>
 	<description>skill tree project</description>
 	<properties>
 		<java.version>17</java.version>
 	</properties>
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-hateoas</artifactId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
any specific reason to use spring-boot-starter-data-jpa

Reply:
Hi @bhtibrewal can you share the reason for this here, I wanted to understand what points to consider while choosing one.

Reply:
as you suggested I read about it and understood that it's a data access layer, it removes the DAO implementation and boilerplate and provides the CRUD operations in a much simplified way.

references I read: https://www.baeldung.com/the-persistence-layer-with-spring-data-jpa
https://spring.io/projects/spring-data-jpa#overview